src/lib.rs

```diff
@@ -759,6 +759,27 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
             }
         }
     }
+
+    /// Get a mutable reference to the underlying data structure
+    ///
+    /// This is marked as unsafe as it allows to invalidate the
```

Comment:
Thanks for the PR. You should probably mention the assumptions that you ths user should not be violating in this comment? Wasn't there something about trailing zeros in storage blocks? IIRC Vob will assume that "unused" bits in the storage are zero.

Reply:
That's correct, and probably a good point. It may also be worthwhile exposing the cleanup function available to the public API, and also allowing to set the length.

Reply:
Hrm, I'm not sure about that. Best wait for @ltratt to comment.

Reply:
FWIW, the `Vec` library also has `set_len()` and `as_mut_ptr`, that basically do the same.